spec/api/v1/openapi.rb

```diff
@@ -15,6 +15,7 @@ def doc
         {
           openapi: '3.0.3',
           info: info,
+          basePath: "#{Settings.path_prefix}/#{Settings.app_name}",
```

Comment:
Shouldn't the base path include the `v1/` or at least have that included within individual specs?

Reply:
I believe that's set in the [version field](https://github.com/RedHatInsights/compliance-backend/blob/master/spec/api/v1/openapi.rb#L28).